### PR TITLE
Evk v1.3 add a key

### PR DESCRIPTION
**Pull request by YangPiCui**
https://github.com/YangPiCui/ErgonomicVerticalKeyboard/tree/master/EVK_v1.3
I added two additional keys on the handwired/evk/v1_3 keyboard. I also adjusted the default keymap slightly.

## Description

The physical keyboard has more keys than the firmware

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ X ] Keyboard (addition or update)
- [ X ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ X ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ X ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ X ] I have added tests to cover my changes.
- [ X ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by YangPiCui** [comment]
Make some more changes

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/evk/v1_3/config.h [hunk 1]
```diff
@@ -39,13 +39,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
 #define MATRIX_ROW_PINS \
-    { B0, B1, B2, B3, B7, D0 }
+    { B0, B1, B2, B3, B7, D0 }  // row 0, 1, 2, ...
```
Comment (fauxpark) [inline keyboards/handwired/evk/v1_3/config.h:42]:
```suggestion
    { B0, B1, B2, B3, B7, D0 }
```

File: keyboards/handwired/evk/v1_3/config.h [hunk 2]
```diff
@@ -39,13 +39,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 #define MATRIX_COL_PINS \
-    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }
+    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }  // column 0, 1, 2, 3, ...
```
Comment (fauxpark) [inline keyboards/handwired/evk/v1_3/config.h:44]:
```suggestion
    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/evk/v1_3/config.h (hunk 2 above)
Comment (YangPiCui) [reply keyboards/handwired/evk/v1_3/config.h:44]: Hi, would it be possible to retain these comments to assist readers in the future?  
https://github.com/YangPiCui/ErgonomicVerticalKeyboard/tree/master/EVK_v1.3/ElectronicsFirmwareAndSoftware#121-handwiredevkv1_3

File: keyboards/handwired/evk/v1_3/config.h (hunk 1 above)
Comment (YangPiCui) [reply keyboards/handwired/evk/v1_3/config.h:42]: Hi, would it be possible to retain these comments to assist potential readers?  
https://github.com/YangPiCui/ErgonomicVerticalKeyboard/tree/master/EVK_v1.3/ElectronicsFirmwareAndSoftware#121-handwiredevkv1_3

Comment (fauxpark) [reply keyboards/handwired/evk/v1_3/config.h:42]: I'm not sure what value it adds, it should be fairly self-explanatory.

Comment (YangPiCui) [reply keyboards/handwired/evk/v1_3/config.h:42]: Alright.

**Review by fauxpark — APPROVED** [review]

**Comment by YangPiCui** [comment]
Hi, the Travis CI Build failed. What can I do to remedy this?

**Comment by YangPiCui** [comment]
Hi, do you know why the layout in the QMK Configurator does not match that in the keymap.c file?
https://config.qmk.fm/#/handwired/evk/v1_3/LAYOUT
https://github.com/qmk/qmk_firmware/blob/master/keyboards/handwired/evk/v1_3/keymaps/default/keymap.c

**Pull request merged**